webui/react/src/components/kit/InputNumber.tsx

```diff
@@ -0,0 +1,17 @@
+import { InputNumber as AntdInputNumber } from 'antd';
+import React from 'react';
+
+interface InputNumberProps {
+  className?: string;
+  defaultValue?: number;
+  disabled?: boolean;
+  min?: number;
+  onChange?: () => void;
+  precision?: number;
+  value?: number;
+}
```

Comment:
question: the original props have more useful props https://4x.ant.design/components/input-number/#API
Are there reasons why we limit these, such as `max`, `status` etc (not just for `InputNumber`, other components too)

---

question: I havent tested it, but i think we cannot use HTML/React native props, such as `ref`, `dir` etc. Is it intentional?

Reply:
Added `max` and `step` to `InputNumber`. As we move forward with switching our codebase to these components we can have a larger discussion about which props are necessary.